### ISSUE TITLE
Syntax error at runtime

### ISSUE THREAD

**Issue by mvisonneau** [trainline/envmgr-cli#1]
I was unable to get it working following the doc:

```
~$ pip install envmgr-cli
~$ export ENVMGR_HOST=***
~$ export ENVMGR_USER=***
~$ export ENVMGR_PASS=***
~$ envmgr
invalid syntax (asg.py, line 50)
~$ envmgr --help
invalid syntax (asg.py, line 50)
```

Is there something that I missed ?

**Comment by mvisonneau** [comment]
Ok found it, looks like it not compatible with Python 3.x. Might be worth to update the README :)

**Comment by duncanhall** [comment]
Thanks, I'm working on compatibility for both Python 2 and 3 which will hopefully be available soon.

**Comment by duncanhall** [comment]
This is **fixed**  :white_check_mark: as of [1.3.0](https://github.com/trainline/envmgr-cli/releases/tag/1.3.0)
